harness/determined/cli/checkpoint.py

```diff
@@ -123,7 +123,28 @@ def delete_checkpoints(args: Namespace) -> None:
         bindings.delete_DeleteCheckpoints(cli.setup_session(args), body=delete_body)
         print("Deletion of checkpoints {} is in progress".format(args.checkpoints_uuids))
     else:
-        print("Aborting deletion of checkpoints.")
+        print("Stopping deletion of checkpoints.")
+
+
+@authentication.required
+def checkpoints_file_rm(args: Namespace) -> None:
+    if (
+        args.yes
+        or len(args.glob) == 0
+        or render.yes_or_no(
+            "All files matching specified globs will be deleted in all checkpoints provided\n"
+            "in checkpoint storage. Do you still want to proceed?"
+        )
+    ):
+        c_uuids = args.checkpoints_uuids.split(",")
+        remove_body = bindings.v1CheckpointsRemoveFilesRequest(
+            checkpointGlobs=args.glob,
+            checkpointUuids=c_uuids,
+        )
+        bindings.post_CheckpointsRemoveFiles(cli.setup_session(args), body=remove_body)
+        print(f"Removal of files from checkpoints {args.checkpoints_uuids} is in progress")
```

Comment:
I believe the only way to wait for checkpoint gc now is to poll `/tasks` like we do in checkpoint tests now, which feels somewhat hacky trying to catch the timing. Since delete checkpoint doesn't have a wait option would a follow up ticket be okay? 

I think we do need to overall improve checkpoint gc visibility to the user somehow.